Gruntfile.js

```diff
@@ -73,6 +73,10 @@ module.exports = function(grunt) {
       tinymce: {
         // copy the mobile font file
         command: 'cp node_modules/tinymce/skins/ui/oxide/fonts/tinymce-mobile.woff web/app/css/tinymce/fonts/tinymce-mobile.woff'
+      },
+      ove: {
+        // copy open vector editor umd files
+        command: 'cp node_modules/@deltablot/open-vector-editor-umd/* web/app/ove/'
```

Comment:
isn't it possible to include it in the webpack bundle?

Reply:
Maybe, I don't know. There are several files in that folder. Next to the .css and .js there are also some fonts (.woff, .woff2, .ttf, .eot). Will they be handled by webpack?

Reply:
Yes webpack can handle fonts. Css can be added during the SCSS build step.